crates/wasi/src/preview2/host/logging.rs

```diff
@@ -0,0 +1,13 @@
+use crate::preview2::{bindings::logging::logging, WasiView};
+
+impl<T: WasiView> logging::Host for T {
```

Comment:
Yeah, I can do that. It's convenient for now, because with stderr being a host stream that gets buffered, it's extra important for debugging to have some kind of debug facilty that can dump text directly to the host stderr. But when this PR is closer to landing I can split it out.

Reply:
Hm that seems like an issue with our stderr design then as well, that should definitely be suitable for debugging. We don't have any writes though that we don't push out via tokio though right? Is the issue that it's a slightly deferred write rather than immediate?

Reply:
I've now removed the logging code.